### DISCUSSION TITLE
Examples in docs (Spectrum) unclear

### DISCUSSION THREAD

**Discussion by ivanjeremic** [adobe/react-spectrum#3829] [General]
Examples like this one one the [https://react-spectrum.adobe.com/react-spectrum/DialogContainer.html#visual-options](https://react-spectrum.adobe.com/react-spectrum/DialogContainer.html#visual-options) Are not very helpful because copy pasting that code does not work, where do `<Edit/>` & `<EditDialog/>` come from where is the rest? Examples should ideally work just by copy pasting the examples.

**Comment by reidbarber** [comment]
`Edit` and `EditDialog` are defined in previous examples on the page. This was just done to avoid being repetitive. We could definitely improve this to be more clear though.
